build.config.ts

```diff
@@ -1,14 +1,12 @@
 import { basename } from 'node:path'
 import { defineBuildConfig } from 'unbuild'
-import fg from 'fast-glob'
+import { globSync } from 'tinyglobby'
 
 export default defineBuildConfig({
-  entries: [
-    ...fg.sync('src/commands/*.ts').map(i => ({
-      input: i.slice(0, -3),
-      name: basename(i).slice(0, -3),
-    })),
-  ],
+  entries: globSync(['src/commands/*.ts'], { onlyFiles: true, absolute: false, expandDirectories: false }).map(i => ({
+    input: i.slice(0, -3),
+    name: basename(i).slice(0, -3),
+  })),
```

Comment:
`onlyFiles` and `absolute` are set to their default values here, which is redundant i think?

```suggestion
  entries: globSync(['src/commands/*.ts'], { expandDirectories: false }).map(i => ({
    input: i.slice(0, -3),
    name: basename(i).slice(0, -3),
  })),
```

Reply:
thx